docs/grid/custom_filter.md

```diff
@@ -4,6 +4,124 @@ Sylius Grids come with built-in filters, but there are use-cases where you need
 
 To add a new filter, we need to create an appropriate class and form type.
 
+## Using attributes
+
+{% code title="src/Grid/Filter/SuppliersStatisticsFilter.php" lineNumbers="true" %}
+```php
+<?php
+
+namespace App\Grid\Filter;
+
+use App\Form\Type\Filter\SuppliersStatisticsFilterType;
+use Sylius\Bundle\GridBundle\Doctrine\DataSourceInterface;
+use Sylius\Component\Grid\Attribute\AsFilter;
+use Sylius\Component\Grid\Filtering\FilterInterface;
+
+#[AsFilter(
+    formType: SuppliersStatisticsFilterType::class,
+    type: 'suppliers_statistics',
+    template: 'templates/Grid/Filter/suppliers_statistics.html.twig'
```

Comment:
Sylius 2.0 and stack uses new file structure for templates dir with lowercase names.

```suggestion
    template: 'templates/grid/filter/suppliers_statistics.html.twig'
```

Is `supplier statistics` related to dashboard or some other page? I thinking of what better path will suite it, rather than root `/grid/filter/`.